acceptance: list all containers

tbg opened the pull request:
It looks like the implementation of ContainerList in docker/engine-api/client
expects -1 to be passed for listing all containers. This could fix #6906.

<!-- Reviewable:start -->

---

This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/6946)

<!-- Reviewable:end -->

tamird commented on acceptance/cluster/docker.go:
```diff
@@ -326,7 +326,10 @@ func (cli resilientDockerClient) ContainerCreate(
-		containers, cerr := cli.ContainerList(ctx, types.ContainerListOptions{All: true})
+		containers, cerr := cli.ContainerList(ctx, types.ContainerListOptions{
+			All:   true,
+			Limit: -1, // no limit
```
can you point to some code here? I think that'd be useful

tamird commented:
LGTM

tbg committed:
acceptance: list all containers

It looks like the implementation of ContainerList in docker/engine-api/client
expects -1 to be passed for listing all containers. This could fix #6906.

tschottdorf commented:
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

*[acceptance/cluster/docker.go, line 331 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/6946#-KImtZugyQItm6rMge61-r1-331:-KJ2180-Vuss2VJFBLqL:-499146009) ([raw file](https://github.com/cockroachdb/cockroach/blob/7f9c2138e7d4397d613e859459e1914f4c202971/acceptance/cluster/docker.go#L331)):*
<details><summary><i>Previously, tamird (Tamir Duberstein) wrote…</i></summary>
> can you point to some code here? I think that'd be useful
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/6946)*
<!-- Sent from Reviewable.io -->
